.gitignore

```diff
@@ -32,3 +32,16 @@ DerivedData
 # CocoaPods
 Pods
 *.lock
+
+.idea/.name
+.idea/encodings.xml
+.idea/misc.xml
+.idea/modules.xml
+.idea/runConfigurations/iOS.xml
+.idea/runConfigurations/Tests.xml
```

Comment:
You might want to just add

```
.idea/
```

To your global gitignore so you don't have to add it to every gitignore. http://stackoverflow.com/a/22885996/717416